tests/sqlparser_snowflake.rs

```diff
@@ -2846,3 +2846,17 @@ fn test_parse_show_columns_sql() {
     snowflake().verified_stmt("SHOW COLUMNS IN TABLE abc");
     snowflake().verified_stmt("SHOW COLUMNS LIKE '%xyz%' IN TABLE abc");
 }
+
+#[test]
+fn test_projection_with_nested_trailing_commas() {
+    let sql = "SELECT a FROM b, LATERAL FLATTEN(input => events)";
+    let _ = snowflake().parse_sql_statements(&sql).unwrap();
+
+    //Single nesting
+    let sql = format!("SELECT ({sql})");
+    let _ = snowflake().parse_sql_statements(&sql).unwrap();
+
+    //Double nesting
+    let sql = format!("SELECT ({sql})");
+    let _ = snowflake().parse_sql_statements(&sql).unwrap();
```

Comment:
Can we instead write out the sql explicitly? I think that makes it easier to figure out what is being tested, and easier to change if needed going forward

Reply:
👍